sdcm/cluster.py

```diff
@@ -1909,7 +1911,9 @@ def install_scylla(self, scylla_repo):
                 self.remoter.run('sudo apt-get install -y openjdk-8-jre-headless -t jessie-backports')
                 self.remoter.run('sudo update-java-alternatives -s java-1.8.0-openjdk-amd64')
             elif self.is_debian9():
+                force = ''
```

Comment:
If you're removing debian 8 support you can remove the elif above as well

Reply:
nope, cause if someone tries to run using debian8, it must not enter in here, as it will not succeed anyway.